user/build.xml

```diff
@@ -147,14 +147,125 @@
   <target name="compile-jakarta" description="Compile all class files for jakarta servlets"
           unless="compile.complete">
     <sequential>
-      <copy todir="${project.build}/jakarta-src">
-        <fileset dir="src" />
+      <copy todir="${project.build}/jakarta-src" encoding="${javac.encoding}">
```

Comment:
looks like this needs to be closed? i'm not sure how it built for you with this missing

Reply:
My local build was before I merged it via ui. Sorry. I fixed it.